client/app/dashboard/model/list/ModelList.js

```diff
@@ -716,6 +716,9 @@ angular.module('dashboard.Dashboard.Model.List', [
       $scope.pagingOptions.pageSize = $scope.pagingOptions.pageSize.toString();
       $scope.loadItems();
     }
+    if (newVal && oldVal && newVal.pageSize != oldVal.pageSize) {
+      $scope.pagingOptions.currentPage = 1;
```

Comment:
Would this be better between in the if block above it, and before the loadItems()?